### PR TITLE
Update Index endpoint URL

### PR DESCRIPTION
**Pull request by jhunzik**

### CODE REVIEW

File: rest/specs/src/main/filtered-resources/openapi/openapi.yaml [hunk 1]
```diff
@@ -22,7 +22,7 @@ servers:
   - url: ${rest.server.url}
 
 paths:
-  /product/{productId}/cst:
+  /product/{productId}:
```
Comment (emmberk) [inline rest/specs/src/main/filtered-resources/openapi/openapi.yaml]: ✏️ `/index/{productId}:`

File: rest/specs/src/main/filtered-resources/openapi/openapi.yaml [hunk 2]
```diff
@@ -191,7 +191,7 @@ components:
         path:
           type: string
           description: The path for the REST request that generated this error.
-          example: /store/product/00067360b70e4acfab561fe593ad3f7a/cst
+          example: /store/product/00067360b70e4acfab561fe593ad3f7a
```
Comment (emmberk) [inline rest/specs/src/main/filtered-resources/openapi/openapi.yaml]: ✏️`example: /index/00067360b70e4acfab561fe593ad3f7a`

File: rest/specs/src/main/filtered-resources/openapi/openapi.yaml [hunk 3]
```diff
@@ -22,7 +22,7 @@ servers:
   - url: ${rest.server.url}
```
Comment (emmberk) [inline rest/specs/src/main/filtered-resources/openapi/openapi.yaml]: ✏️ Remove `DefaultError` below

Comment (emmberk) [inline rest/specs/src/main/filtered-resources/openapi/openapi.yaml]: ❗️Don't forget to bump the version to `0.2.0-SNAPSHOT`

File: rest/specs/src/main/filtered-resources/openapi/openapi.yaml (hunk 1 above)
Comment (jhunzik) [reply rest/specs/src/main/filtered-resources/openapi/openapi.yaml]: Wouldn't this make it less resource-based semantically speaking? What about `/product/{productId}/index`?

Comment (emmberk) [reply rest/specs/src/main/filtered-resources/openapi/openapi.yaml]: The store API will still have a store metadata endpoint. That service is the one that needs to be "resource-based".

**Pull request merged**